test/createAzureResource.test.ts

```diff
@@ -120,6 +120,25 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.state, 'Running', `Function App state should be 'Running' rather than ${createdApp.state}.`);
     });
 
+    test('Add new setting', async () => {
+        let applicationSettingsValue: string | undefined;
+        const applicationSettings: { key: string; value: string } = { key: 'FUNCTION_EXTENSION_URL', value: "https://github.com/microsoft/vscode-azurefunctions" };
+        const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
+        assert.ok(createdApp, `Function App ${resourceName1} not found`);
+        ext.ui = new TestUserInput([resourceName1, applicationSettings.key, applicationSettings.value]);
+        await vscode.commands.executeCommand('azureFunctions.appSettings.add');
+        const listapplicationSettings: WebSiteManagementModels.StringDictionary = await webSiteClient.webApps.listApplicationSettings(resourceName1, resourceName1);
+        assert.ok(listapplicationSettings.properties, 'Failed to get application settings');
+        if (listapplicationSettings.properties !== undefined) {
+            if (listapplicationSettings.properties.FUNCTION_EXTENSION_URL !== undefined) {
+                applicationSettingsValue = listapplicationSettings.properties.FUNCTION_EXTENSION_URL;
+            } else {
+                assert.ok(listapplicationSettings.properties.FUNCTION_EXTENSION_URL, `The key "${applicationSettings.key}" by the application setting not found.`);
+            }
+        }
+        assert.equal(applicationSettingsValue, applicationSettings.value, `The application setting for "${applicationSettings.key}=${applicationSettings.value}" not found`);
```

Comment:
Rather than the logic above, just do

`assert.equal(listApplicationSettings.properties[applicationSettings.key], applicationSettings.value)